### PR TITLE
DS-1254 search config

### PR DESCRIPTION
**Pull request by froboy**
https://yusa.atlassian.net/browse/DS-1254
- Update Google Search and Search API modules with config pages

To test:
- enable `openy_google_search` and `openy_search_api`
- Check `/admin/openy/settings/google-search`, try changing a setting, and save
- Check `/admin/openy/settings/search-api`, try changing a setting, and save
- See that the docs link to up-to-date places

### CODE REVIEW

**Comment by froboy** [comment]
@podarok for review

File: openy_search/openy_google_search/openy_google_search.info.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
-name: Open Y Google Search
+name: YMCA Website Services Google Search
 type: module
-description: Provides a Google Custom search Engine for Open Y.
+description: Provides a Google Custom search Engine for YMCA Website Services.
 core_version_requirement: ^8.8 || ^9 || ^10
-package: Open Y
+package: YMCA Website Services
 version: 8.x-1.0
```
Comment (podarok) [inline openy_search/openy_google_search/openy_google_search.info.yml:6]:
```suggestion
version: 1.1.0
```

**Review by podarok — APPROVED** [review]
Small things

**Comment by froboy** [comment]
@podarok thanks. Updated.

**Pull request merged**